test/middleware/service.availability.middleware.spec.ts

```diff
@@ -35,15 +35,7 @@ describe("service availability middleware tests", () => {
     expect(response.text).not.toContain("Service offline - Register an overseas entity");
   });
 
-  test("update disabled should return service offline page", async () => {
-    mockIsActiveFeature.mockReturnValueOnce(false).mockReturnValueOnce(false);
-    const response = await request(app).get("/update-an-overseas-entity");
-
-    expect(response.text).toContain("Service offline - Register an overseas entity");
-  });
-
-  test("update enabled should not return service offline page", async () => {
-    mockIsActiveFeature.mockReturnValueOnce(false).mockReturnValueOnce(true);
+  test("update journey should not return service offline page", async () => {
```

Comment:
Do we still need to test around this?

Reply:
Similar tests are already present, this test just checks that '/update-an-overseas-entity' route behaves correctly and doesn't return the service offline page.